package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@hawk.so/webpack-plugin",
-  "version": "0.0.10",
+  "version": "0.0.11",
```

suspected comings is a new feature, so

```suggestion
  "version": "0.1.0",
```